docs/en/observability/cloud-monitoring/azure/monitor-azure-agent.asciidoc

```diff
@@ -361,6 +358,8 @@ change and start sending Azure activity logs to {es}.
 [[azure-elastic-agent-visualize-azure-logs]]
 === Step 5: Visualize Azure activity logs
 
+//TODO: Need to go back and fix the places where I've changed this by mistake to Discover.
+
```

Comment:
@mdbirnstiehl How do we want to handle updates on this page? It's not just a simple text replacement. Should I just leave the updates for you? Or maybe Arianna?

Reply:
Nevermind. I am just going to direct users to Discover and remove the screenshot of the Streams view.